src/Extensions/SwiftypeSiteTreeCrawlerExtension.php

```diff
@@ -18,21 +19,51 @@
  */
 class SwiftypeSiteTreeCrawlerExtension extends SiteTreeExtension
 {
+
+    /**
+     * Urls to crawl
+     *
+     * array keyed by getOwnerKey
+     *
+     * @var array
+     */
+    private $urlsToCrawl = [];
```

Comment:
@blueo can you please make this `protected`, rather than `private`? Allows folks to extend this class if they wish.

Reply:
hmmm my feeling is that is what the get/set methods are for ie the api to the property? you could extend those fairly easily?